### PR TITLE
Adding more Group badges/options

### PR DESCRIPTION
**Pull request by devinmatte**
People are proud of their groups. Display more on the profiles of people who have earned them, and show more as options.

This adds 3da's as a badge

And adds 3das, and webmasters to the dropdown list

### CODE REVIEW

**Comment by mbillow** [comment]
What the hell is a VRA?

**Comment by stevenmirabito** [comment]
@mbillow Virtual Reality Admin 😉

**Comment by mbillow** [comment]
Excuse me... is that even a thing?!

**Comment by devinmatte** [comment]
@mbillow https://profiles.csh.rit.edu/group/vra

**Comment by mbillow** [comment]
@stevenmirabito @devinmatte Please someone give me a single reason to not delete that group... which is literally just Tino.

**Comment by devinmatte** [comment]
Shitposts aside. Tino wanted to establish a group to ensure that the VR equipment is maintained and managed by a group. Especially if we want to continue to make those resources available to all members, and as we'll likely get even more freshman that are interested in VR.

**Comment by mbillow** [comment]
@devinmatte We need to formalize a process for these random groups on floor that just spring out of nowhere... Drink Admins, 3DAs, Keymasters, etc. I don't like the fact that Tino can just call himself a "VRA" and somehow have control over something all of house owns.

**Comment by devinmatte** [comment]
@mbillow I don't see it as generally a bad thing that these groups appear. In fact I think all it does is pass down responsibilities and tribal knowledge more effectively. By labeling someone as a group, we can look back in profiles and see all the old alumni that used to manage that area. As for the VRA thing, yeah maybe right now it's not super useful being only Tino, but going forward I think it makes a lot of sense for there to a VR admins group because right now that's a job RTPs don't really want to do, that doesn't have to be RTP work. I also think someone should be in charge of it. Currently that's been Tino whether there's an ldap group for it not. It doesn't need to be in my PR, but I don't see an issue with the group existing

**Comment by mbillow** [comment]
@stevenmirabito Are you planning on using the current `keymaster` group in your implementation. It makes adding it here useless if not.

File: profiles/templates/profile.html [hunk 1]
```diff
@@ -73,6 +77,14 @@ <h6 class="card-subtitle mb-2 text-muted">
 							<span class="badge badge-pill badge-primary">Drink Admin</span>
 							{% endif %}
 
+                            {% if "vra" in member_info.group_list %}
+							<span class="badge badge-pill badge-vr">Virtual Reality Administrator</span>
+							{% endif %}
+
+                            {% if "keymaster" in member_info.group_list %}
+							<span class="badge badge-pill badge-purple">Gatekeeper Admins</span>
```
[the remark this answers is not in the document]
Comment (devinmatte) [reply profiles/templates/profile.html]: I was keeping them consistent with their naming in ldap.
In the dropdown I kept them consistent with them all being 'Admins'

[the remark this answers is not in the document]
Comment (devinmatte) [reply profiles/templates/profile.html]: I feel like just renaming VR 'Administrator' to VR 'Admin' fixes this problem

**Pull request merged**